source/system/Numbers.ooc

```diff
@@ -137,6 +137,7 @@ SizeT: cover from size_t extends ULLong
 SSizeT: cover from ssize_t extends LLong {
 	toString: func -> String { "%u" formatSSizeT(this) }
 }
+PtrDiff: cover from ptrdiff_t extends SSizeT
```

Comment:
What I mean is that `extend` indicates a `is-a` relation (or maybe I got that part wrong), so is should be possible to use the result type in any place where `Big` is expected ? and they have different sizes... or does it work differently for covers ?